src/content/en/2019/fonts.md

```diff
@@ -0,0 +1,121 @@
+
+**06.46: Uses a Google Fonts link**
+
+While Google Fonts’ dominance on requests was very impressive, it’s very noteworthy that only 29% of pages included a Google Fonts `<link>` element. This could mean a few things:
+
+- When pages uses Google Fonts, they use _a lot_ of Google fonts. They are free, after all. This seems like the most likely reason.
+- A lot of people are using Google Fonts with `@import` instead of `<link>`. My hunch is that this seems unlikely but maybe!
+- Google Fonts are being requested using an [HTTP `Link: ` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link) instead. Now we’re going off the deep end into very unlikely scenarios.
+
+**06.47: Uses a Google Fonts link first**
+
+Google Fonts documentation encourages the `<link>` for the Google Fonts CSS to be placed as the first child in the `<head>` of a page. This is a big ask! In practice, this is not common as only half a percent of all pages took this advice. More so, if a page is using `preconnect` or `dns-prefetch` (read on for more about these) as `<link>` elements, these would come before the Google Fonts CSS anyway. In conclusion, 🤷‍♂️!
+
+**06.01:  Are fonts being hosted on the same host or by a different host than the main content?**
+
+This differentiation is important for performance reasons, increasingly relevant in an H2 world where the performance gap between a same-host and different-host connection can be wider. Same-host requests can also be better prioritized against same-host requests for other resources in the waterfall. Recommendations to mitigate this include `preconnect`, `dns-prefetch`, and `preload` but high priority web fonts (those used by a lot of content on your page) should be same-host requests to minimize the performance impact of web fonts.
+
+This is perhaps unsurprising given Google Fonts dominance in `06.02`, as 75% Google Fonts requests-dominance would all be different host requests. Note that `preload` is not yet available with Google Fonts.
+
+**06.03:  MIME types**
+
+WOFF2 is pretty well supported in web browsers today. Google Fonts serves WOFF2, a format that offers improved compression over its predecessor. To me this is perhaps supporting evidence for a case to be made to go WOFF2-only for web fonts. I wonder where the double-digit WOFF usage is coming from—perhaps Internet Explorer?
+
+Third place `octet-stream` (and `plain` a little further down) would seem to suggest that a lot of web servers are misconfigured to not properly send a MIME type with web font file requests.
+
+**06.36–38:  src formats included in @font-face blocks**
+
+I was hoping to see [SVG fonts](https://caniuse.com/#feat=svg-fonts) on the decline. They’re buggy and implementations have been removed from every browser except Safari. Time to drop these, y’all.
+
+**06.39–41:  src formats combinations**
+
+- [ ] TODO *Zach’s note: The order here is important, yeah.*
```

Comment:
@zachleat are you responding to this comment: https://docs.google.com/a/google.com/spreadsheets/d/108g6LXdC3YVsxmX1CCwrmpZ3-DmbB8G_wwgQHX5pn6Q/edit?disco=AAAACyHTXnc

To say that you'd prefer them in the original declaration order rather than alphabetical sorting?